ibis/pandas/execution/temporal.py

```diff
@@ -44,6 +44,14 @@ def execute_extract_timestamp_field_series(op, data, **kwargs):
     return getattr(data.dt, field_name).astype(np.int32)
 
 
+@execute_node.register(ops.ExtractEpoch, (pd.Timestamp, pd.Series))
+def execute_epoch(op, data, **kwargs):
+    return pd.Series(
+        (pd.DatetimeIndex(data) - pd.Timestamp('1970-01-01'))
+        // pd.Timedelta('1s')
+    )
```

Comment:
thanks @datapythonista do you know if the result is the same? or could it have any  round difference?

Reply:
The internal representation is the number of nanoseconds since epoch. Didn't check, but I assume the `.astype(int)` is returning that internal representation without errors. I think the `//` will ignore second decimals, not round to the closer. But among both implementations, if there are differences, I'd go with my implementation, since it's taking the epoch directly, not with an intermediary delta. But I'd say it should be exactly the same.